features/step_definitions/logging.rb

```diff
@@ -904,6 +904,23 @@
   })
   raise "Error subscript amqstreams" unless @result[:success]
 
+  workers = BushSlicer::Node.get_labeled("kubernetes.io/os=linux,node-role.kubernetes.io/worker=", user: admin)
+  step %Q/I run the :debug admin command with:/, table(%{
+    | resource     | node/#{workers.first.name} |
+    | oc_opts_end  |                            |
+    | exec_command | chroot                     |
+    | exec_command | /host                      |
+    | exec_command | fips-mode-setup            |
+    | exec_command | --check                    |
```

Comment:
Try use `oc get machineconfigs` and check whether the output contains `99-master-fips`, which should be easier and faster than using `oc debug node`
I get the idea from https://github.com/openshift/verification-tests/blob/master/features/step_definitions/node.rb#L8-L24

Reply:
Updated, thanks for your info, really helpful.